File: INICIANTE/JAVASCRIPT/1011.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift(); };

var raio = prompt("raio")
var PI = 3.14159;
var volume = parseFloat(4/3 * PI * Math.pow(raio, 3));

console.log("VOLUME = " + volume.toFixed(3));

File: INICIANTE/JAVA/1011.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        double r, v;
        Scanner sc = new Scanner(System.in);
        double pi = 3.14159;
        r = sc.nextDouble();
        v = (4.0 / 3) * pi * Math.pow(r, 3.0);
        System.out.println(String.format("VOLUME = %.3f", v));
    }
} 
